packages/open-api-gateway/.log4jsrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Same here - maybe best to generate this as part of `private/projects/open-api-gateway-project.ts`?

Reply:
No need for this since it's source code now :)